<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="bot_txt">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>361</width>
     <height>341</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(187, 211, 255);</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="player_txt">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>70</y>
     <width>361</width>
     <height>341</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(187, 211, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="botname_lbl">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Bot</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_txt">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>40</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="message_lbl">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>450</y>
     <width>551</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Battle begins!</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="row_line">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>160</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="col_line">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>160</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="shoot_btn">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Shoot!</string>
   </property>
  </widget>
  <widget class="QLabel" name="scores_bot_lbl">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Scores: 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="scores_player_lbl">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Scores: 0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
